.github/workflows/build-and-test.yml

```diff
@@ -659,21 +659,49 @@ jobs:
   publish-stable:
     runs-on: ubuntu-24.04
     needs: [lint, unittest, integration-tests]
+    env:
+      RELEASE_NOTES: release-notes.md
     if: startsWith(github.ref, 'refs/tags/v') && github.repository == 'open-telemetry/opentelemetry-collector-contrib'
     permissions:
       contents: write
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
-      - name: Set Release Tag
+        with:
+          fetch-depth: 0
+
+      - run: npm install simple-git@^3.27.0
+
+      - name: Set current and previous tags
         id: github_tag
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: ./.github/workflows/scripts/set_release_tag.sh
+
       - name: Prepare release notes
         env:
           RELEASE_TAG: ${{ steps.github_tag.outputs.tag }}
+          NEW_CONTRIBUTOR_TEXT: ${{ steps.new-contributor-text.outputs.result }}
```

Comment:
Why is this defined as an environment variable?

Reply:
good catch, this is a leftover and can be removed